crates/abq_cli/tests/cli.rs

```diff
@@ -4842,10 +4842,15 @@ fn write_partial_rwx_v1_json_results_on_early_runner_termination() {
         "kind": "RSpec",
         "language": "Ruby"
       },
+      "otherErrors": [
+        {
+          "message": "ABQ had an error communicating with the native runner: early eof at crates/abq_runners/generic_test_runner/src/lib.rs@1035:87"
+        }
+      ],
       "summary": {
         "canceled": 0,
         "failed": 0,
-        "otherErrors": 1,
+        "otherErrors": 2,
```

Comment:
🤔 should this be 1?

Reply:
or differently stated, we only have one `otherError` in the list of other errors and I the summary is supposed to reflect those lists (other errors and the tests)